Misc/NEWS.d/next/Core and Builtins/2021-02-15-15-06-43.bpo-40170.ZYeSii.rst

```diff
@@ -0,0 +1,3 @@
+:c:func:`PyIter_Check` macro is now a function, in order to hide implementation
+details. The macro accessed :c:member:`PyTypeObject.tp_iternext` directly.
```

Comment:
Well. It's complicated. It was declared both as a macro **and** a function. I suggest:

```suggestion
:c:func:`PyIter_Check` is now always declared as a function, in order to hide implementation
details. The macro accessed :c:member:`PyTypeObject.tp_iternext` directly.
```

Reply:
Yes, that's better.